rotkehlchen/exchanges/binance.py

```diff
@@ -553,17 +549,20 @@ def _api_query_dict_within_time_delta(
             raise AssertionError(f'Unexpected binance method case: {method}.')
 
         results: List[Dict[str, Any]] = []
+
         # Create required time references in milliseconds
+        start_ts = Timestamp(start_ts * 1000)
```

Comment:
I was thinking this is "typingly" wrong... it should be `TimestampMS`, and probs I'd get a warning about reusing the same name for different types. Same for `end_ts`.

Reply:
no it's not. These types are made up. And I actually don't like them much. I think we should not be having made up types such as Timestamp e.t.c.

```python
T_Timestamp = int
Timestamp = NewType('Timestamp', T_Timestamp)
```

Initially I put them thinking the type system should be as granular as possible and try and type everything. But at least like this sometimes when you want to add a price and an int you get a problem and you need to satisfy the type gods.

The jury is still out on this one.